aea/common.py

```diff
@@ -18,4 +18,13 @@
 # ------------------------------------------------------------------------------
 """This module contains the common types and interfaces used in the aea framework."""
 
+from typing import Any, Dict, List, Optional, Union
+
+
 Address = str
+
+Primitive = Union[str, int, bool, float]
+_JSONDict = Dict[Any, Any]  # temporary placeholder
+_JSONList = List[Any]  # temporary placeholder
```

Comment:
It would be nice if eventually we had an implementation like this: https://github.com/python/mypy/issues/731#issuecomment-539905783